analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -553,7 +553,11 @@ class Pip(
                 id = Identifier(
                     type = "PyPI",
                     namespace = "",
-                    name = dependency["package_name"].textValue(),
+                    // Normalize all PyPI package names to be lowercase and hyphenated as per PEP 426:
+                    // "All comparisons of distribution names MUST be case insensitive,"
+                    // and MUST consider hyphens and underscores to be equivalent.
+                    // https://www.python.org/dev/peps/pep-0426/#name
+                    name = dependency["package_name"].textValue().toLowerCase().replace('_', '-').replace('.', '-'),
```

Comment:
There is a look-up in using the identifier via a `Map` in line 355 which is case sensitive.
Thus, I believe the package name assigned by `parsePipShowOutput()` needs to also be normalized in order to not break the lookup.